db/migration/src/lib.rs

```diff
@@ -64,12 +64,20 @@ impl MigratorTrait for MigratorIndex {
             Box::new(m20240226_000004_index_create_node_info::Migration),
             Box::new(m20240226_000005_index_create_chain_info::Migration),
             Box::new(m20240226_000006_index_create_network_eligibility::Migration),
-            Box::new(m20240226_000008_node_create_settings::Migration),
```

Comment:
You cannot delete migrations like that since this will require other nodes to re-create that db. Instead you should create a new migration which removes the tables etc in question. Moreover, how is the data from the old table transferred to the new table?

Reply:
The The GlobalSettings table wasn't used anywhere until now, so no data to transfer. Then for the same argument the `create_settings` migration can be moved as, because we split it into a different db, it will anyway require a db re-sync I guess.

Reply:
No, let's create a completely separate DB inside the hoprd package, this functionality has nothing to do with the lower level hopr-lib